saas/models.py

```diff
@@ -1340,14 +1357,22 @@ class AbstractRole(models.Model):
         help_text=_("Key to identify the grant of the role"))
     extra = get_extra_field_class()(null=True,
         help_text=_("Extra meta data (can be stringify JSON)"))
-
+    ends_at = models.DateTimeField(null=True, blank=True,
```

Comment:
Run `python manage.py makemigrations --name v0_17_2 saas` to also make the migrations file (v0_17_2 because it is the version of the next release per [saas/__init__.py](https://github.com/djaodjin/djaodjin-saas/blob/master/saas/__init__.py)).